src/store/modules/progress.js

```diff
@@ -259,6 +266,13 @@ const getters = {
     }
   },
 
+  complete_hideout: (state) => (id) => {
```

Comment:
Minor thing: You may have thoughts on this, and if so I'd love to hear them, but I've been trying to name things `{noun}_{verb}` to indicate a getter (hideout_complete? quest_complete?) and `{verb}_{noun}` to indicate a mutator (complete_quest, complete_hideout, uncomplete_objective). This is the first time I've used a datastore in this style, so there may be other conventions you're used to, and if so I'm happy to keep it as is, otherwise I'd just be interested in keeping it conform

Reply:
Alright, I will rename it. I don't care either way as long as it's consistent. Didn't notice the pattern.